daemon/cluster/cluster.go

```diff
@@ -280,27 +280,30 @@ func (c *Cluster) GetDataPathAddress() string {
 	return ""
 }
 
-// GetRemoteAddress returns a known advertise address of a remote manager if
+// GetRemoteAddressList returns the known advertise address of a remote managers if
```

Comment:
`the known advertise address of a remote managers` => `the known advertise addresses of remote managers`?